packages/web-components/fast-components-msft/src/tree-item/tree-item.styles.ts

```diff
@@ -183,6 +183,11 @@ export const TreeItemStyles = css`
         left: var(--expand-collapse-button-nested-width, calc(${heightNumber} * -1px));
     }
 
+    :host(.leaf) .content-region {
```

Comment:
FYI in general we should avoid setting classes on the `:host` - the reason for this is interop with UI frameworks where classNames may be data-bound - see https://github.com/microsoft/fast/issues/3389 for an example.

Not a huge issue here because this component *already* adds classes, but good to be aware of going forward